### ISSUE TITLE
Color scheme picker 

### ISSUE THREAD

**Issue by RaeldZues** [rabobank-cdc/DeTTECT#66]
Request add for a color scheme picker for the output of the ATTA&CK coverage map

**Comment by marcusbakker** [comment]
Good idea. We already have been thinking about this for a while. Our current idea is to integrate this in the Editor and store the colour preference inside the YAML file, which will automatically influence the output generated by the CLI.
